### PR TITLE
Agent invitations

### PR DESCRIPTION
**Pull request by AhmadSaleem**

### CODE REVIEW

File: app/controllers/agents/invitations_controller.rb [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 
   def update
-    @agency = self.resource.owned_agency
     super
     if resource.invitation_accepted_at?
-      resource.wholesalers << resource.invited_by.wholesaler
-      resource.agency_applications.update_all(agency_id: @agency.id) if resource.agency_applications.present?
-      UserMailer.welcome(resource.email).deliver_later
-      CompanyOwnerMailer.agent_invite_accept(resource).deliver_later
+      if resource.invited_by.is_a?(Agent)
```
Comment (colinambler) [inline app/controllers/agents/invitations_controller.rb:6]: Could we do this logic through a concern or inheriting somehow?  Looks strange and feels unorganized that wholesaler logic is in agents controller.

File: app/controllers/agents/registrations_controller.rb [hunk 2]
```diff
@@ -15,4 +15,8 @@ def sign_up_params
     def account_update_params
       params.require(:agent).permit(:first_name, :last_name, :email, :password, :password_confirmation, :current_password, :profile_picture, :profile_picture_cache)
     end
+
```
Comment (colinambler) [inline app/controllers/agents/registrations_controller.rb]: should this be to all user roles?

Comment (colinambler) [reply app/controllers/agents/registrations_controller.rb]: nvm, I see you did it

File: app/views/dashboard/agency/agent_invitations/_actions.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="btn-group">
```
Comment (colinambler) [inline app/views/dashboard/agency/agent_invitations/_actions.html.erb]: would love to see this whole employee invitation process get refactored into one partial, used for both Wholesaler and Agency

File: app/views/dashboard/insurance_applications/index.html.erb [hunk 4]
```diff
@@ -20,13 +20,17 @@
               <tbody>
               <% @applications.each do |application| %>
                 <tr>
-                  <td><%= application.insurance_application.template.title %></td>
-                  <td><%= application.agent.full_name if application.agent.present? %></td>
+                  <td><%= application.application_title %></td>
+                  <% if application.agent.present? %>
```
Comment (colinambler) [inline app/views/dashboard/insurance_applications/index.html.erb]: could be rewritten as ternary to consolidate into one line

File: app/controllers/agents/invitations_controller.rb (hunk 1 above)
Comment (ghost) [reply app/controllers/agents/invitations_controller.rb:6]: i think it could be done with concerns.

**Pull request merged**